src/connectors/withHistoryActions.spec.jsx

```diff
@@ -40,14 +41,19 @@ describe('connectors/withHistoryActions', () => {
     actions.forEach((action) => {
       it(`should call ${action}`, () => {
         const pathname = 'PATHNAME';
-
+        const options = {
```

Comment:
By adding those options here we loose regression testing (for backwards compatibility). I'll adjust the tests.